### DISCUSSION TITLE
DHCP needed for on-site connections

### DISCUSSION THREAD

**Discussion by marccolemont** [Ysurac/openmptcprouter#2700] [Q&A]
I'm trying to configure either a RUTX12 or Pi4 to be used for Livestreaming on-site. Meaning one of the WAN connections need to be plugged at the venue internet connection. No way even thinking about in changing their IP DHCP settings or IP range.
Do I need to place an extra router in between the local internet connection and the openMPTCrouter?
As I read the WAN's need to be 192.168.10.1 / 192.168.11.1 etc. Or can we set DHCP instead on the WAN, so it gets an IP from the local internet connection?

**Comment by Ysurac** [comment]
DHCP can be set on a WAN but this need to be a dedicated port, so possible on a RUTX12 but on a PI4 need to have a ethernet USB key (on a powered USB hub). VLAN can also be used but sometimes doesn't give a good result.
As platform, a x86_64 barebone with AES support give often a good result (and can be cheaper than a RPI4 now...).
